railties/lib/rails/api/task.rb

```diff
@@ -9,14 +10,21 @@ class Task < RDoc::Task
             README.rdoc
             lib/active_support/**/*.rb
           ),
-          exclude: "lib/active_support/vendor/*"
+          exclude: [
+            'lib/active_support/i18n_railtie.rb',
+            'lib/active_support/xml_mini/*'
+          ]
         },
 
         "activerecord" => {
           include: %w(
             README.rdoc
             lib/active_record/**/*.rb
-          )
+          ),
+          exclude: [
+            'lib/active_record/tasks/*',
```

Comment:
Ugh, it looks there are all apart from `DatabaseTasks` which is still present on my local version.

* https://github.com/rails/rails/blob/773e45a59163415e8bc247ebee1663c7e8113c81/activerecord/lib/active_record/tasks/mysql_database_tasks.rb#L3
* https://github.com/rails/rails/blob/773e45a59163415e8bc247ebee1663c7e8113c81/activerecord/lib/active_record/tasks/sqlite_database_tasks.rb#L3
* https://github.com/rails/rails/blob/773e45a59163415e8bc247ebee1663c7e8113c81/activerecord/lib/active_record/tasks/postgresql_database_tasks.rb#L3